.github/workflows/pr_label_check.yml

```diff
@@ -13,24 +13,24 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Check aspect label
-        uses: sugarshin/required-labels-action@v0.3.2
+        uses: agilepathway/label-checker@v1.2.9
         with:
-          required_oneof: "💻 aspect: code,📄 aspect: text,🤖 aspect: dx,🕹 aspect: interface,♿️ aspect: a11y"
+          any_of: "💻 aspect: code,📄 aspect: text,🤖 aspect: dx,🕹 aspect: interface,♿️ aspect: a11y"
 
   check_goal_label:
     name: Check goal label
     runs-on: ubuntu-latest
     steps:
       - name: Check goal label
-        uses: sugarshin/required-labels-action@v0.3.2
+        uses: agilepathway/label-checker@v1.2.9
         with:
-          required_oneof: "🌟 goal: addition,🛠 goal: fix,✨ goal: improvement,🧰 goal: internal improvement"
+          any_of: "🌟 goal: addition,🛠 goal: fix,✨ goal: improvement,🧰 goal: internal improvement"
 
   check_priority_label:
     name: Check priority label
     runs-on: ubuntu-latest
     steps:
       - name: Check priority label
-        uses: sugarshin/required-labels-action@v0.3.2
+        uses: agilepathway/label-checker@v1.2.9
         with:
-          required_oneof: "🟩 priority: low,🟨 priority: medium,🟧 priority: high,🟥 priority: critical"
+          any_of: "🟩 priority: low,🟨 priority: medium,🟧 priority: high,🟥 priority: critical"
```

Comment:
```suggestion
          one_of: "🟩 priority: low,🟨 priority: medium,🟧 priority: high,🟥 priority: critical"
```

Reply:
I agree with you @sarayourfriend, good catch!